Gemfile

```diff
@@ -1,4 +1,5 @@
 source 'https://rubygems.org'
+ruby "2.4.1"
```

Comment:
This is exactly the intention. Stop using different development environments in each of the developer machines and enforce the use of a single virtual development environment that is as close as possible to the _production_ environment.

The advantages are:

* Avoid possible conflicts that arise from configuration differences between the different development machines.
* Allow developers to start working on the project as son as possible by automating dependencies installation and configuration.

Also, is a recommended setting from Heroku.